app/models/publication-flow.js

```diff
@@ -46,57 +45,4 @@ export default class PublicationFlow extends Model {
     }
     return `${this.publicationNumber}`;
   }
-
-  get translationRequestsTotal() {
-    return this.publicationService.getPublicationCounts(this.id).then((totals) => {
```

Comment:
opkuis van al deze getters nog niet compleet,
Er zijn nog references in pods/publications/index/template.hbs
https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/3e37a30cc19c9882e8e2db4254a63750ae28e030/app/pods/publications/index/template.hbs#L189